src/components/compose.jsx

```diff
@@ -1298,8 +1328,8 @@ function Compose({
               <label class="toolbar-button">
                 <input
                   type="file"
-                  accept={supportedMimeTypes.join(',')}
-                  multiple={mediaAttachments.length < maxMediaAttachments - 1}
+                  accept={supportedMimeTypes?.join(',')}
+                  multiple={!(maxMediaAttachments - mediaAttachments.length < 2)}
```

Comment:
Honestly this feels difficult to read for me. If this is meant for handling undefined `maxMediaAttachments`, I would rather code this as `maxMediaAttachments ? … : …`. But if `maxMediaAttachments` is undefined, is it really ok to set this `multiple=true`? 🤔

Reply:
I think "no limit" is not a good sensible default; it's like saying you can attach unlimited files 😂

Is there really no max limit set at all from Pleroma/Akkoma? Perhaps another API call to know the limits?

Reply:
> it's like saying you can attach unlimited files 😂

Yeah, that's exactly what it means!

The only limit on uploads is on individual file size, not on number, so no limit is exposed.

Same goes for the MIME types: you can attach any kind of file in a post, so no list of acceptable MIME types is made available. :smile: